README.md

```diff
@@ -1,5 +1,7 @@
 # Vroom: Launch vim tests
 
+![Alt text](/images/vroom_logo.png?raw=true "VRoom Logo")
```

Comment:
"Alt text" is pretty terrible alt text. The alt text should be "Vroom Logo". I don't think this needs any title text, so just move that over.